4、异常
 - 两种捕获异常的方法：
    - 用try...catch表达式将会抛出异常的函数括起来
    - 把函数调用包含在catch表达式里
    try FuncOrExpressionSequence of
        Pattern1 [when Guard1] -> Expression1;
        Pattern2 [when Guard2] -> Expression2;
        ...
    catch
        ExceptionType: ExPattern1 [when ExGuard1] -> ExExpressions1;
        ExceptionType: ExPattern2 [when ExGuard2] -> ExExpressions2;
        ...
    after
        AfterExpressions
    end
    - ExceptionType是原子throw、exit、error中的一个，默认值是throw。
    